core/cmd_exec.py

```diff
@@ -1421,7 +1424,7 @@ class Tracer(object):
     - argv and span ID of the SimpleCommand that corresponds to that
     - then print line number using arena
     - set -x doesn't print line numbers!  OH but you can do that with
-      PS4=$LINENO 
```

Comment:
Huh, this time I didn't do an auto-indent intentionally, so I must have an auto-PyLint tool set up on my Vim.